src/main/java/com/example/v0/demo/controller/ToDoController.java

```diff
@@ -0,0 +1,52 @@
+package com.example.v0.demo.controller;
+
+import com.example.v0.demo.model.ToDo;
+import com.example.v0.demo.service.ToDoService;
+import com.example.v0.demo.service.ToDoServiceImpl;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/todos")
+public class ToDoController {
+    private final ToDoServiceImpl toDoService;
+    public ToDoController(ToDoServiceImpl toDoService){//Constructor por obra y gracia del espitiru santo
+        this.toDoService = toDoService;
+    }
+    /*@GetMapping
+    public List<To Do> getAllToDos(){
+        return toDoService.findAll();// obtener toodo
+    }*/
+    @GetMapping
+    public ResponseEntity<List<ToDo>> getAllToDos(){
+        List<ToDo> todos = toDoService.findAll();
+        return ResponseEntity.ok(todos);
+    }
+
+    @PostMapping
+    public ResponseEntity<ToDo> createToDo(@RequestBody ToDo todo){
+        ToDo createToDo = toDoService.add(todo);
+        return new ResponseEntity<>(createToDo, HttpStatus.CREATED);//se crea la mafufada esta, osea new to do
```

Comment:
?? you should not use that kind of comments on a PR, comments are just for meaningful things or documentation